<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN">
<html lang="en">

  <head>
    <title>biocommons (gh)</title>
    <link rel="stylesheet" type="text/css" href="styles.css">
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
  </head>

  <body>
    
    <script type="text/javascript">
      $(document).ready(function () {
        $.ajaxSetup ({
          cache: false,
          crossDomain: true
        });
      });
      function stuff_nissues(repo_slug) {
      	 var base_url = "https://api.bitbucket.org/1.0/repositories/biocommons"
         $.ajax({
           url: base_url + "/" + repo_slug + "/issues",
           type: "GET",
           data: "limit=0;status=new;status=open",
           async: false,
           dataType: 'JSONP',
           contentType: "application/json",
           success: function(response) {$("#nissues-"+repo_slug).text(response["count"])}
         })
      }
    </script>
    

    <h1>About</h1>

    <p>biocommons
    (<a href="http://biocommons.org/">http://biocommons.org/</a>) is a
    collection of software tools related in some way to sequence
    analysis, and particularly genome sequence analysis.  Most of the
    tools were originally written by a few colleagues
    at <a href="https://invitae.com/">Invitae</a>.  We hope that
    you'll use and contribute to these tools.</p>

    <p>Names like "biocommons" are, well, common. You may also wish to
    see the other unaffiliated sites with similar
    names: <a href="http://www.bio-commons.org/">Bio-Commons</a> (with
    a
    hyphen), <a href="http://www.cbs.umn.edu/learn/undergraduates/cbssb/biocommons">UMN
    biocommons</a>,
    or <a href="http://biology.wisc.edu/biocommons.htm">UW
    BioCommons</a>.</p>


    <h1>Packages</h1>

    <!-- 
	 TODO: add # of commits, # of commiters, and # of issues to summary 
	 e.g., https://api.bitbucket.org/1.0/repositories/biocommons/hgvs/issues?limit=0&status=open&status=new
      -->

    <dl>
      
      <dt>
	<a href="https://bitbucket.org/biocommons/bioutils">bioutils</a>
        [<a href="https://bitbucket.org/biocommons/bioutils/issues?status=new&status=open"><span class="count" id="nissues-bioutils"></span> Issues</a>]
	<script type="text/javascript">
	  $(document).ready(function () { stuff_nissues("bioutils") })
	</script>
		
      </dt>
      <dd>
	<p>This package provides some simple tools and lookup tables used primarly by the hgvs and uta packages.</p>
      </dd>
      
      <dt>
	<a href="https://bitbucket.org/biocommons/eutils">eutils</a>
        [<a href="https://bitbucket.org/biocommons/eutils/issues?status=new&status=open"><span class="count" id="nissues-eutils"></span> Issues</a>]
	<script type="text/javascript">
	  $(document).ready(function () { stuff_nissues("eutils") })
	</script>
		
      </dt>
      <dd>
	<p>eutils is a Python package dedicated to searching for, fetching, and parsing records from NCBI using their E-utilities interface.</p>
      </dd>
      
      <dt>
	<a href="https://bitbucket.org/biocommons/hgvs">hgvs</a>
        [<a href="https://bitbucket.org/biocommons/hgvs/issues?status=new&status=open"><span class="count" id="nissues-hgvs"></span> Issues</a>]
	<script type="text/javascript">
	  $(document).ready(function () { stuff_nissues("hgvs") })
	</script>
		
      </dt>
      <dd>
	<p>This package provides a Python library to facilitate the use of genome, transcript, and protein variants that are represented using the Human Genome Variation Society (Mutnomen) recommendations.</p>
      </dd>
      
      <dt>
	<a href="https://bitbucket.org/biocommons/multifastadb">multifastadb</a>
        [<a href="https://bitbucket.org/biocommons/multifastadb/issues?status=new&status=open"><span class="count" id="nissues-multifastadb"></span> Issues</a>]
	<script type="text/javascript">
	  $(document).ready(function () { stuff_nissues("multifastadb") })
	</script>
		
      </dt>
      <dd>
	<p>MultiFastaDB presents a collection of indexed fasta files as a single source. The intent is to simplify accessing a virtual database of sequences that is distributed across multiple files.</p>
      </dd>
      
      <dt>
	<a href="https://bitbucket.org/biocommons/uta">uta</a>
        [<a href="https://bitbucket.org/biocommons/uta/issues?status=new&status=open"><span class="count" id="nissues-uta"></span> Issues</a>]
	<script type="text/javascript">
	  $(document).ready(function () { stuff_nissues("uta") })
	</script>
		
      </dt>
      <dd>
	<p>The UTA (Universal Transcript Archive) stores transcripts aligned to sequence references (typically genome reference assemblies). It supports aligning the same transcript to multiple references using multiple alignment methods.</p>
      </dd>
      
      <dt>
	<a href="https://bitbucket.org/biocommons/uta-align">uta-align</a>
        [<a href="https://bitbucket.org/biocommons/uta-align/issues?status=new&status=open"><span class="count" id="nissues-uta-align"></span> Issues</a>]
	<script type="text/javascript">
	  $(document).ready(function () { stuff_nissues("uta-align") })
	</script>
		
      </dt>
      <dd>
	<p>uta-align provides C-based Needleman-Wunsch and Smith-Waterman alignment algorithms with a Python interface.</p>
      </dd>
      
    </dl>
    

    <h1>License</h1>
    <p>All of the packages in the biocommons are distributed with the
    Apache License (2.0).</p>


  </body>

</html>
