# Welcome to the biocommons!

## Mission

The biocommons accelerates research, promotes data sharing, and improves human health by
collaborating on interoperable and high-quality bioinformatics software and data, primarily  for
sequence variant interpretation, and to foster a community that develops these tools.

## About us

- [Mission](mission.md): Why we exist (or longer executive summary?)
- [Culture](culture.md): How we treat each other respectfully
- [Governance](governance.md): How we organize to promote effective collaboration
- [Practices](practices.md) and [Tools](tools.md): How we get stuff done coherently
- [Architecture](architecture.md) and [Coding Guidelines](coding-guidelines.md): How we build interoperable tools with consistent behavior
- [Community](community.md): How you can get involved, learn, use, and contribute joyfully


## Organization

TBW

### Project Teams

- [anyvar](https://github.com/biocommons/anyvar)
  [in development] Proof-of-Concept variation translation, validation, and registration service
- [bioutils](https://github.com/biocommons/bioutils)
  provides common tools and lookup tables used primarily by the hgvs and uta packages
- [eutils](https://github.com/biocommons/eutils) simplified searching, fetching, and parsing records
  from NCBI using their E-utilities interface
- [hgvs](https://github.com/biocommons/hgvs)
  Python library to parse, format, validate, normalize, and map sequence variants.
- [SeqRepo](https://github.com/biocommons/biocommons.seqrepo)
  non-redundant, compressed, journalled, file-based storage for biological sequences
- [seqreo-rest-service](https://github.com/biocommons/seqrepo-rest-service)
  OpenAPI-based REST interface to biological sequences and sequence metadata
- [UTA](https://github.com/biocommons/uta)
  Universal Transcript Archive: comprehensive genome-transcript
  alignments; multiple transcript sources, versions, and alignment
  methods; available as a docker image
- [uta-align](https://github.com/biocommons/uta-align)
  C-based Needleman-Wunsch and Smith-Waterman alignment algorithms with a Python interface.
